cmd/fdo_server/server.go

```diff
@@ -161,6 +161,7 @@ func (s *Server) Start() error {
 		return err
 	}
 	defer func() { _ = lis.Close() }()
+	// TODO(runcom): we don't listen on external address?
```

Comment:
The external address represents "how will a device connect to this service?" It could be an FQDN or IP (either public or private, depending on the deployment scenario). This is then used in the rendezvous info for the device credential (I'm guessing).